src/theme.dark.less

```diff
@@ -37,6 +37,10 @@
   }
 }
 
+.header-logo {
+  background: #0000;
```

Comment:
you've used `@white` in another place, can you use `@black` here?

Reply:
This is transparent. But yes, I can use a variable like everywhere else